app/src/main/res/layout/charity_item_horizontal.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+
+    <!--<LinearLayout-->
+    <!--android:layout_width="match_parent"-->
+    <!--android:layout_height="wrap_content"-->
+    <!--android:orientation="vertical"-->
+    <!--android:padding="0dp">-->
+    <ImageView
+        android:id="@+id/ivCharityImage"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
```

Comment:
I see you use 16dp and 8dp multiple times. You can reuse this by declaring it in the `dimens.xml`

you can call 16dp as `standard_padding` or `standard_margin` and 8dp as `small_padding` or `small_margin` in the `dimens.xml` and then reference them here